packages/bpk-component-loading-button/src/BpkLoadingButton-test.tsx

```diff
@@ -16,9 +16,7 @@
  * limitations under the License.
  */
 
-/* @flow strict */
```

Comment:
Hey Ollie could you please explain the removals of `flow strict` and the changings of `toMatchSnapshot` ?

Reply:
As this file uses Typescript having flow in here does nothing and flow isn't installed in the repo anymore for a long time.

In terms of the snapshots, moved away from these as they aren't reliable and to use a more reliable testing over snapshots, something we advocate for in Skyscanner